### DISCUSSION TITLE
Quarkus + Hibernate + Trino + Cassandra Connector

### DISCUSSION THREAD

**Discussion by jaiprasad266** [trinodb/trino#12137] [Q&A] [answered]
Hi ,

We are trying to fetch data from trino for the cassandra connector using the quarkus rest endpoint , facing multiple exceptions like dialect not available .

Can anyone help us in using quarkus + hibernate with trino .

Please comment.
<img width="558" alt="cassandra_trino_config" src="https://user-images.githubusercontent.com/54102261/165266677-6b57b25e-61b8-4765-bd04-262bcf485bfd.PNG">

**Comment by ebyhr** [comment, marked as answer]
I would recommend asking in Hibernate community. Relates to https://github.com/hibernate/hibernate-orm/pull/3496
